### ISSUE TITLE
Suggestion - reduce amount of support generated based on new printer option for maximum bridge length

### ISSUE THREAD

**Issue by cgapeart** [alexrj/Slic3r#1925]
Using either 1.0.0, or 1.1.0, I am fighting with general issues about support material.  There are existing issues discussing how much support is generated and the distance from perimeter shells already, I don't want to duplicate that.  What I wanted to suggest is that an option (either per-printer, per-extruder, or in the job profile).  The main case would be to cover internal infill.  If the user has calibrated their printer and can specify what the maximum bridge is, Slic3r could eliminate some internal support material.  Where a gap is bigger than the maximum bridge, then pillar style support could be added to support the top without adding too much support.

I have some samples to attach - just not available until later today.  One part I had trouble on was http://www.thingiverse.com/thing:240547 -- though it's not a good example because it will print just fine without support if you rotate the STL so it prints on it's side with the pocket arches facing +z.

In short, I don't want support material around small holes, and large holes or pockets could probably get away with one or two central pillars instead of a complete infill job that can't be removed without breaking the part.
